README.md

```diff
@@ -4,12 +4,64 @@ Shared CKEditor 4 GitHub workflows.
 
 ## How it works?
 
-The main function of `setup-workflows.yml` workflow is to propagate common workflows and keep them up to date.
+The main function of this repository is to propagate common workflows and keep them up to date.
 
-The `setup-workflows.yml` workflow checkouts this repository once a day in the repository it is executed and updates (or creates) all common workflows. Common workflows are stored in `./workflows/` directory. There is also `setup-workflows.yml` workflow there, which means, once it is set it will auto-update itself too.
+The `setup-workflows.yml` workflow checkouts this repository once a day in the target repository and updates (or creates) all common workflows. Common workflows are stored in `workflows/` directory here and are copied to `.github/workflows/` directory in a target repository. There is also `setup-workflows.yml` workflow there, which means, once it is set it will auto-update itself too.
 
 ## Setup
 
-1. Copy `workflows/setup-workflows.yml` workflow to your repository.
+1. Copy `workflows/setup-workflows.yml` workflow to your repository as `.github/workflows/setup-workflows.yml`.
 1. Setup `secrets.GH_BOT_USERNAME` and `secrets.GH_BOT_EMAIL` to GitHub user which has a push access to your repository.
-1. Setup `secrets.GH_WORKFLOWS_TOKEN` to GitHub token which has `workflows` permission.
+1. Setup `secrets.GH_WORKFLOWS_TOKEN` to GitHub token which has `write` and `workflows` permissions.
+
+## Optional configuration
+
+Some workflows may be altered by configuration options (refer to [Available workflows](#available-workflows) section below). The configuration file is optional and not present by default. If needed, it should be added to any repository using common workflows as `.github/workflows-config.json` file.
+
+## Available workflows
+
+### setup-workflows
+
+This is the main workflow responsible for propagating all common workflows. When it is added to any other repository it checkouts this repository and copies all common workflows from `workflows/` directory to `.github/workflows/` one. It is done once a day so any changes are propagated on a daily basis. See `workflows/setup-workflows.yml` file.
+
+It is a cron job task so will be triggered only on main repository branch.
```

Comment:
Maybe we could add information when cron runs the action? (`02:00 UTC`).

We could also add this information to other actions.